Bigtable/src/Instance.php

```diff
@@ -0,0 +1,331 @@
+     *
+     * This method may require a service call.
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function info(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Check if the instance exists.
+     *
+     * This method requires a service call.
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function exists(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Fetch a fresh representation of the instance from the service.
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/
+     *     google.bigtable.admin.v2#google.bigtable.admin.v2.GetInstanceRequest GetInstanceRequest
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/
+     *     google.bigtable.admin.v2#instance Instance
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function reload(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Return the instance state.
+     *
+     * When instances are created or updated, they may take some time before
+     * they are ready for use. This method allows for checking whether an
+     * instance is ready.
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function state(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Bigtable\BigtableClient;
+     * $bigtable = new BigtableClient();
+     * $operation = $instance->create(
+     *     [$bigtable->buildClusterMetadata('my-cluster', 'my-location', null, 3)]
+     * );
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/
+     *     google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array[] $clusterMetadataList Use {@see Google\Cloud\Bigtable\BigtableClient::buildClusterMetadata()}
+     *        to create properly formatted cluster configurations.
+     * @param array $options [optional] {
+     *        Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *           [Using labels to organize Google Cloud Platform resources]
+     *           (https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
```

Comment:
another link which won't render right. links need to be on a single line. (this overrides line length limitations)